gettsim/interface.py

```diff
@@ -11,6 +10,7 @@
 from gettsim.dag import create_function_dict
 from gettsim.dag import execute_dag
 from gettsim.dag import prune_dag
+from gettsim.functions_loader import convert_paths_to_internal_functions_to_imports
```

Comment:
Not sure about the name of that function.

`get_imports_from_paths` 

?

Reply:
Fixed in #198.